src/containers/Layout/MainLayout.tsx

```diff
@@ -12,13 +14,13 @@ const MainLayoutWrapper = styled.div`
   background-color: var(--color-bg-main);
 
   .main {
-    height: 100vh;
+    height: calc(100vh - 29px);
```

Comment:
Unfortunately this is needed because of how the current layout uses fixed heights... Thankfully when we introduce the new layout we won't have to do stuff like this 😅